core/layout_api.php

```diff
@@ -561,13 +561,15 @@ function layout_navbar_button_bar() {
 	if( $t_show_report_bug_button )  {
 		$t_bug_url = string_get_bug_report_url();
 		echo '<a class="btn btn-primary btn-sm" href="' . $t_bug_url . '">';
-		echo '<i class="fa fa-edit"></i> ' . lang_get( 'report_bug_link' );
+		print_icon( 'fa-edit');
+		echo lang_get( 'report_bug_link' );
 		echo '</a>';
 	}
 
 	if( $t_show_invite_user_button ) {
 		echo '<a class="btn btn-primary btn-sm" href="manage_user_create_page.php">';
-		echo '<i class="fa fa-user-plus"></i> ' . lang_get( 'invite_users' );
+		print_icon( 'fa-fa-user-plus');
```

Comment:
Should be
`print_icon( 'fa-user-plus');`

Reply:
OK